.shippable.yml

```diff
@@ -25,6 +25,7 @@ build:
         options: "-e HOME=/home/buildslave --privileged=true --tty --net=bridge --user buildslave"
 
     ci:
+      - sudo pip3 install psutil
```

Comment:
why isn't psutil in scripts/requirements.txt?

Reply:
@andrewboie it has been added in another PR, I add here to make sure CI runs well and marked DNM

Reply:
I see it now, it's already in master. OK.

Reply:
@andrewboie but I think we can merge this PR right now.

Reply:
not yet, docker image needs to be updated first and this commit needs to be removed.